public/static/docs/command-reference/get.md

```diff
@@ -12,8 +12,8 @@ into the current working directory.
 usage: dvc get [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
 
 positional arguments:
-  url         Location of DVC project or Git repository to download from
-  path        Path to a file or directory within the project or repository
+  url   Location of DVC project or Git repository to download from.
```

Comment:
usually more spaces here?

Reply:
This is to keep in a single line:

![image](https://user-images.githubusercontent.com/1477535/73569710-8c3abc80-4430-11ea-90d4-e508adbc4a71.png)

Reply:
but we already have a few commands with two lines? this format makes it somewhat harder to read I would say ... minor issue though

Reply:
Yeah there's no exact rule about how many spaces we leave for these. Probably it's not very consistent. Since you feel there's not enough spaces I'll just add a few more haha and break the lines... ⌛️

Reply:
OK done in 6f4de01f. I also updated a few more cmd refs for more consistent spacing (just up to index.md for now)

Reply:
I'm extracting this into the next regular updates PR #966